src/commands/BootstrapCommand.js

```diff
@@ -1,13 +1,12 @@
 import FileSystemUtilities from "../FileSystemUtilities";
-import NpmUtilities from "../NpmUtilities";
 import PackageUtilities from "../PackageUtilities";
+import BootstrapMixin from "../mixins/BootstrapMixin";
 import Command from "../Command";
 import async from "async";
-import find from "lodash.find";
 import path from "path";
 import semver from "semver";
 
-export default class BootstrapCommand extends Command {
+export default class BootstrapCommand extends BootstrapMixin(Command) {
```

Comment:
Yes, probably.

The benefit of a mixin here is its methods get attached to the command.  They have access to everything the command has with no special initialization to worry about.

What about mixins bothers you?